app/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/GetRecordsConnector.scala

```diff
@@ -47,18 +47,13 @@ class GetRecordsConnector @Inject() (
       if (isHawk) s"${appConfig.hawkConfig.getRecordsUrl}/$eori/$recordId"
       else s"${appConfig.pegaConfig.getRecordsUrl}/$eori/$recordId"
 
-    val bearerToken =
-      if (isHawk) appConfig.hawkConfig.getRecordBearerToken else appConfig.pegaConfig.getRecordBearerToken
-
-    val forwardedHost = if (isHawk) appConfig.hawkConfig.forwardedHost else appConfig.pegaConfig.forwardedHost
```

Comment:
We may want to remove the getRecordbearerToken from the PagaInstanceConfig, as this is not used anymore